### PR TITLE
Remove deprecated devices table attributes

### PR DESCRIPTION
**Pull request by brec9824**

### CODE REVIEW

File: app/Http/Controllers/ApiController.php [hunk 1]
```diff
@@ -58,11 +58,6 @@ public function update(Request $request)
             'error_msg'     => 'nullable|string',
             'limitsw_open'   => 'nullable|boolean',
             'limitsw_closed' => 'nullable|boolean',
```
Comment (Cyberzoid1) [inline app/Http/Controllers/ApiController.php]: I dont send limit switch data at all to the server.
Not even sending that as sensor data.

File: app/Http/Controllers/ApiController.php [hunk 2]
```diff
@@ -80,11 +75,6 @@ public function update(Request $request)
         $device->error_msg = $request->input('error_msg');
         $device->limitsw_open = $request->input('limitsw_open');
         $device->limitsw_closed = $request->input('limitsw_closed');
```
Comment (Cyberzoid1) [inline app/Http/Controllers/ApiController.php]: Same as above

File: app/Http/Controllers/ApiController.php (hunk 1 above)
Comment (brec9824) [reply app/Http/Controllers/ApiController.php]: Fixed 898bba8

File: app/Http/Controllers/ApiController.php (hunk 2 above)
Comment (brec9824) [reply app/Http/Controllers/ApiController.php]: Fixed 898bba8

**Pull request merged**